index.html

```diff
@@ -1,21 +1,112 @@
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
-    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
 </head>
+
 <body>
+    <!-- navigation bar -->
+    <header>
+        <nav class="navbar">
+            <img class="logo" src="img/karma-logo.svg" alt="Karma Logo">
+            <ul class="links">
+                <li>
+                    <a href="#"><span class="dark">Meet Karma</span></a>
+                </li>
+                <li>
+                    <a href="#">How it Works</a>
+                </li>
+                <li>
+                    <a href="#">Store</a>
+                </li>
+                <li>
+                    <a href="#">Blog</a>
+                </li>
+                <li>
+                    <a href="#">Help</a>
+                </li>
+                <li>
+                    <a href="#">Login</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- SHOWCASE -->
+    <section class="showcase">
+        <h2>
+            Introducing Karma
+        </h2>
+        <h4>
+            Bring WiFi with you, everywhere you go.
+        </h4>
+
+        <button class="button" type="button">
+            Learn More
+        </button>
 
+    </section>
+
+    <!-- SEGMENTS -->
+    <section class="seg">
+        <h2>
+            Everyone needs a little Karma.
+        </h2>
+        <div class="segments">
+            <div class="devices">
+                <img src="img/icon-devices.svg" alt="Devices" class="dev">
+                <h5>
+                    Internet for all devices
+                </h5>
+            </div>
+            <div class="cup">
+                <img src="img/icon-coffee.svg" alt="Coffee" class="coffee">
+                <h5>
+                    Boost your productivity
+                </h5>
+            </div>
+            <div class="fuel">
+                <img src="img/icon-refill.svg" alt="refill" class="nuzzle">
+                <h5>
+                    Pay as You Go
+                </h5>
+            </div>
+        </div>
+    </section>
+
+    <!-- FOOTER -->
+    <footer id="footer">
+        <h6>
+            Join us on
+        </h6>
+        <div class="social">
+            <a href="#">
+                <img src="img/Twitter_logo.png" alt="twitter logo" class="twitter">
+            </a>
+            <a href="#">
+                <img src="img/facebook-circled.png" alt="Facebook logo" class="face">
+            </a>
+            <a href="#">
+                <img src="img/Insta-logo.png" alt="Instagram" class="insta">
+            </a>
+        </div>
+        <p class="copy">
+            &copy; Karma Mobility, Inc.
+        </p>
+        <div class="end"></div>
```

Comment:
Why add an empty div here?
If you want to add spacing, add it to the footer.
example: 
```
footer {
   padding-bottom: 100px;
}
```